.cirrus.yml

```diff
@@ -57,14 +75,31 @@ task:
     - ${CHROOT} chown -R build: .
   <<: *standard_build
 
+freebsd_gce_instance: &freebsd_gce_instance
+  platform: FreeBSD
+  image_project: freebsd-org-cloud-dev
+  # Need to use image_name for now: https://github.com/cirruslabs/cirrus-ci-docs/issues/422
+  disk: 30  # Image is 29GB, VM can't be smaller
+  <<: *standard_gce_instance
```

Comment:
This needs to come earlier, otherwise `disk` from `standard_gce_instance` will overwrite the value above.

Reply:
Thanks :)